conf/modules/post_variant_calling.config

```diff
@@ -34,11 +58,25 @@ process {
         ]
     }
 
+    withName: 'VCFS__SORT'{
+        ext.prefix = { "${meta.id}.norm" }
+        ext.when   = { params.normalized_vcfs }
+        publishDir = [
+            mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/normalized/${meta.id}/" }
+        ]
+    }
+
     withName: 'TABIX_EXT_VCF' {
         ext.prefix = { "${input.baseName}" }
         ext.when   = { params.concatenate_vcfs }
     }
 
+    withName: 'TABIX_VCF' {
+        ext.prefix = { "${input.baseName}" }
+        ext.when   = { params.normalized_vcfs }
+    }
+
```

Comment:
these ones can be combined

Reply:
looking at the module, I think you can actually output the tbi in the same process as the vcf, so no need to spin up an extra process for it

Reply:
I've checked the bcftools_norm module and it needs as inputs vcf and tbi. So I guess, we can't exclude it. Or do you mean to output tbi from variant callers directly? But what could be excluded is tabix at the end after sorting, right? Because tbi is ouput from bcftools norm process and is transferred to bcftools sort, so it should end up with sorted vcf and tbi at the end